installer/deb/scripts/postinst

```diff
@@ -78,7 +78,7 @@ generate_certificate() {
     curl -sL $MKCERT_URL -o mkcert || fatal "mkcert download failed."
     chmod +x mkcert || fatal "mkcert permission change failed."
 
-    # We ensure that CAROOT is set. Otherwise, mkcert installation will fail. 
+    # We ensure that CAROOT is set. Otherwise, mkcert installation will fail.
```

Comment:
great! do you use a .sh formatting tool?

Reply:
Yes, but I don't really like it because it changes a lot of stuff in a way that I found less readable, such as removing the indentation of the switch cases
It's "shell-format" in VSCode